python/example_code/s3/s3_basics/README.md

```diff
@@ -103,6 +97,10 @@ python scenario_getting_started.py
 
 **Create a presigned URL**
 
+This example requires the following additional package:
+
+- Requests 2.24.0 or later
+
 Generates a presigned URL and uses the Requests package to get or 
 put a file in an Amazon S3 bucket. For example, run the following command to get
```

Comment:
in an Amazon S3 bucket. --> into an S3 bucket.

Reply:
fixed